website/i18n/zh/docusaurus-plugin-content-blog/2022/03/02/apisix-integration-graphql-plugin.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "GraphQL 碰撞 API 网关 Apache APISIX，提升 API 领域的安全与性能"
+authors:
+  - name: "江晨炜"
+    title: "Author"
+    url: "https://github.com/Chever-John"
+    image_url: "https://avatars.githubusercontent.com/u/43690894?v=4"
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://avatars.githubusercontent.com/u/97138894?v=4"
+keywords: 
+- Apache APISIX
+- API Gateway
+- GraphQL
+- Ecosystem
+description: 本文介绍了 Apache APISIX 和 GraphQL 的特性，以及如何使用云原生 API 网关 Apache APISIX 代理 GraphQL 请求，并提出解决实际场景痛点的方案。
+tags: [Technology,Ecosystem,Security]
```

Comment:
The `Security` tag is meant for CVE news. Please delete this tag to avoid confusion.